core/src/main/scala/cromwell/core/path/BetterFileMethods.scala

```diff
@@ -494,6 +506,8 @@ object BetterFileMethods {
 
   def currentWorkingDirectory: Path = DefaultPath(better.files.File.currentWorkingDirectory.path)
 
+  def DefaultCharset: Charset = Charset.defaultCharset()
```

Comment:
Question for the reviewer: Would a `val` be more appropriate here? defaultCharset() defaults to the system value and if it can not find it to `UTF-8`. I guess there is no value in calling it more than once, but since I was uncertain I left it as a def.

Reply:
Do we expect it to change its value during the lifetime of the program execution?

Reply:
Probably not. That would be quite uncommon, because it is locale related. I will make it a val.

Reply:
Done.